### ISSUE TITLE
In-place replacement of an image with another

### ISSUE THREAD

**Issue by luke-skywalker** [xournalpp/xournalpp#1229]
It would be nice to have a "replace image" function. This is particularly useful if one is making a presentation, and has a fixed space allocated for pictures.
Instead of:
1. duplicating a slide;
2. deleting the picture in the duplicated slide;
3. inserting a new picture in the duplicated slide;
4. fiddling with resizing until the two pictures occupy the same position (which is never perfect anyway)...

...it would be nicer if one could just:
1. duplicate a slide;
2. right click on the existing picure and have a "replace image" function, the new image occupying the place of the old one.

Something [along these lines](https://support.office.com/en-us/article/replace-or-delete-a-picture-in-microsoft-office-f50c4431-a785-49d1-9ee5-a732947cbf1a).

One could use the "snap to grid" option, but this is suboptimal since the corners of the picture most likely won't end on snap-points.

Apologies in advance if this is already possible and I didn't figure out how to do it. If that's the case, I'd appreciate pointers!
Thanks for keeping this amazing piece of software alive!

**Comment by Technius** [comment]
Not sure how other software handles this, but do you expect the replacing picture to keep the size? What if they have different width/height ratios?

**Comment by luke-skywalker** [comment]
Yeah, I would expect the size to be fixed and the aspect ratio to change. This is the behavior in the various Office suites where the functionality is already implemented.

It's an extremely useful functionality in all those situations in which one would like for example to have a series of screenshots from, say, an application that's being demo'd. All screenshots have the same aspect ratio and the same size, and placing them manually makes them inevitably "jumpy" when one goes through the generated document in presentation mode.
